crossplane.yaml

```diff
@@ -8,7 +8,11 @@ metadata:
     meta.crossplane.io/license: Apache-2.0
 spec:
   crossplane:
-    version: ">=v1.13.2-0"
+    version: ">=v1.14.1-0"
```

Comment:
Should we require higher(latest?) patchset here?

Reply:
Good Catch

Reply:
think its fine because we using `>= ` i enabled renovate for our configuration-* repos that we get these updates automatic